v2/samples/network/v1api20220701/refs/v1api20201101_publicipaddress-bastion.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: network.azure.com/v1api20201101
+kind: PublicIPAddress
+metadata:
+  name: bastionpublicip
```

Comment:
Minor: Rename this to aso-sample-publicip?

Reply:
Given that this is a `ref` specifically for `bastion`, it seems better to me to name it in a way so that it's affiliated with the bastion?